applications/pusher/src/modules/pm_apple.erl

```diff
@@ -79,15 +81,26 @@ get_apns(App, ETS) ->
 
 -spec maybe_load_apns(api_binary(), ets:tid()) -> api_pid().
 maybe_load_apns(App, ETS) ->
-    maybe_load_apns(App, ETS, kapps_config:get_binary(?CONFIG_CAT, <<"apple">>, 'undefined', App)).
+    Config = kapps_config:get_json(?CONFIG_CAT, <<"apple">>, kz_json:new(), App),
+    CertBin = kz_json:get_ne_binary_value(<<"certificate">>, Config),
+    Host = kz_json:get_ne_binary_value(<<"host">>, Config, ?DEFAULT_APNS_HOST),
```

Comment:
I would rather you call `kapps_config:get_ne_binary(?CONFIG_CAT, [<<"apple">>,<<"certificate">>])` for these two parameters.They do the same thing extraction and it's more explicit.